docs/trace-agent/datadog.example.yaml

```diff
@@ -19,6 +19,13 @@ api_key: 1234
 # 
 # # hostname
 # hostname: localhost
+#
+# # The host to listen on for metrics and traces. This is ignored when non-local
```

Comment:
I'd write `The host to listen on for Dogstatsd and traces.` for accuracy

Reply:
Done.